from mysql.connector import Error
from sqlconnec import connect
from printcolor import *


def remove_cve():

    cve = input("Enter CVE-ID (Ex: CVE-2024-1234): ")

    conn = connect()
    cursor = conn.cursor()

    try:
        cursor.execute("DELETE FROM CVE_ID WHERE ID = %s", (cve,))
        conn.commit()
        print_green("CVE-ID removed successfully\n")

    except Error as e:
        print_red("Error:", e)
        conn.rollback()
    finally:
        if conn.is_connected():
            cursor.close()
            conn.close()
